seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/sql/SQLTransform.java

```diff
@@ -49,7 +45,6 @@
 import static org.apache.seatunnel.transform.sql.SQLEngineFactory.EngineType.ZETA;
 
 @Slf4j
-@NoArgsConstructor
 @AutoService(SeaTunnelTransform.class)
```

Comment:
Do we need to keep AutoService? Since there is no `NoArgsConstructor` so the SQLTransform cannot load by ServiceLoader.load

Reply:
Good catch. Let me do some change.

Reply:
Done